packages/card/src/Card.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { css, cx } from '@leafygreen-ui/emotion';
+import { uiColors } from '@leafygreen-ui/palette';
+
+const containerStyle = css`
+  background-color: white;
```

Comment:
Could ya add a transition to the border and box-shadow?

Reply:
what kind of transition?